README.md

```diff
@@ -56,6 +56,21 @@ Add software operating systems, programming languages or libraries which are req
 
 *Write usage instructions, including configuration details, settings or arguments available.*
 
+#### Configuration
```

Comment:
Perhaps we could link to the relevant place in the ArchivesSnake docs somewhere in here?